### PR TITLE
Server is now http2 + Allow resize based on height

### PR DESCRIPTION
**Pull request by Nickinthebox**
### Nouveau comportement

Les nouveaux serveurs d'image sont http2.

[Grace au multiplexage apporté par http2](https://www.cloudflare.com/fr-fr/learning/performance/http2-vs-http1.1/), on n'a plus besoin de séparer les calls sur le serveur (entre img.mapado.net et img1.mapado.net)

Par ailleurs, on permet à présent un redimensionnement basé sur une hauteur (et pas seulement une largeur), ce qui est particulièrement pratique pour un logo affiché sur une ligne dont on connait la hauteur, mais dont la largeur peut varier.

### Tache Clickup

🧑‍🔧

### CODE REVIEW

File: Tests/Units/BuilderTest.php [hunk 1]
```diff
@@ -39,11 +39,15 @@ public function testThumbs()
     {
         $baseUrl = '//img.mapado.net/2014/1/1/my-image';
         $url = $this->manager->buildUrl($baseUrl, 200);
-        $this->assertSame($baseUrl . '_thumbs/200', $url);
+        $this->assertSame($baseUrl . '_thumbs/200-0', $url);
+
+        $baseUrl = '//img.mapado.net/2014/1/1/my-image';
+        $url = $this->manager->buildUrl($baseUrl, 0, 200);
+        $this->assertSame($baseUrl . '_thumbs/0-200', $url);
```
Comment (jdeniau) [inline Tests/Units/BuilderTest.php:46]: Can you add a test with `null` and `null` for width and height ? and another one with `['allowwebp' => true]` ?

**Review by jdeniau — APPROVED** [review]
